Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticList.tsx

```diff
@@ -216,30 +240,39 @@ export const DiagnosticList: React.FC<IDiagnosticListProps> = ({ diagnosticItems
     setColumns(staticColumns);
   }, [diagnosticItems]);
 
-  const sortFactor = columns[1].isSortedDescending ? 1 : -1;
-  const displayedDiagnosticItems = diagnosticItems.sort((a, b) => {
-    const aName = getSkillName(a.projectId);
-    const bName = getSkillName(b.projectId);
-    if (aName < bName) {
-      return sortFactor;
-    } else if (aName > bName) {
-      return -sortFactor;
-    } else {
-      return a.location < b.location ? sortFactor : -sortFactor;
-    }
-  });
+  const displayedDiagnosticItems = useMemo(() => {
+    const sortFactor = columns[1].isSortedDescending ? 1 : -1;
+    return diagnosticItems.sort((a, b) => {
+      // Error before Warning
+      const severityComparator = a.severity.localeCompare(b.severity);
+      if (severityComparator === 0) {
+        // Sort by name
+        return sortFactor * getProjectName(a.projectId).localeCompare(getProjectName(b.projectId), locale);
+      }
+      return severityComparator;
+    });
+  }, [diagnosticItems, columns]);
 
   return (
-    <DetailsList
-      isHeaderVisible
-      checkboxVisibility={CheckboxVisibility.hidden}
-      columns={columns}
-      css={detailList}
-      items={displayedDiagnosticItems}
-      layoutMode={DetailsListLayoutMode.justified}
-      selectionMode={SelectionMode.single}
-      setKey="none"
-      onRenderDetailsHeader={onRenderDetailsHeader}
-    />
+    <ScrollablePane>
```

Comment:
why do you need scrollablepane around detailslist

Reply:
https://github.com/microsoft/fluentui/issues/1146 This is the exact issue I face here.
![Screen Shot 2021-06-21 at 4 48 58 PM](https://user-images.githubusercontent.com/13004779/122841319-991fb080-d2b0-11eb-94fb-91bc2d6a4f2c.png)